core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1615,6 +1615,7 @@ public String toString() {
       .build();
   public static final PropertyKey MASTER_RPC_PORT =
       new Builder(Name.MASTER_RPC_PORT)
+          .setAlias("alluxio.master.rpc.port")
```

Comment:
I found an inconsistency that we have key `alluxio.job.master.rpc.port` but for master it is `alluxio.master.port`. 
@calvinjia let me know what you think about this change (I'm not strong on this)
maybe alternatively, we can use ``alluxio.master.port` and `alluxio.job.master.port`

Reply:
My $.02: break backcompat here and add the aliases. I think it is much better to user `*.rpc.port` and `*.web.port`. `*.port` isn't quite descriptive enough

Reply:
I'm in favor of option 3 as well.

Reply:
-> option 3